server/app.js

```diff
@@ -3,11 +3,38 @@ import express, { json, urlencoded } from "express";
 import { join } from "path";
 import cookieParser from "cookie-parser";
 import logger from "morgan";
+import mongoose from "mongoose";
+// var mongoose = require('mongoose');
 
 import indexRouter from "./routes/index";
 import pingRouter from "./routes/ping";
+let  users= require('./schemas/users');
```

Comment:
stay consistent, use import statement (and it should be const either way instead of let)